flow/Trace.cpp

```diff
@@ -1254,7 +1254,7 @@ void TraceBatch::dump() {
 		g_traceLog.writeEvent(buggifyBatch[i].fields, "", false);
 	}
 
-	g_traceLog.flush();
+	onMainThreadVoid([](){ g_traceLog.flush(); }, NULL);
```

Comment:
I realize that I suggested writing it this way initially, but I guess we should probably use `nullptr` instead:

```suggestion
	onMainThreadVoid([](){ g_traceLog.flush(); }, nullptr);
```